src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -366,6 +366,7 @@ public static class Agent
                 public static readonly string ContainerMapping = "agent.containermapping";
                 public static readonly string ContainerNetwork = "agent.containernetwork";
                 public static readonly string Diagnostic = "agent.diagnostic";
+                public static readonly string EnableAdditionalMaskingRegexes = "agent.enableadditionalmaskingregexes";
```

Comment:
This is just moved to keep alphabetical as instructed in comment.

Separate question: this appears to be unused.

Reply:
the second flag is missing here.